rogue-thi-app/lib/backend-utils/mobility-utils.js

```diff
@@ -123,7 +125,7 @@ export function renderMobilityEntry (kind, item, maxLen, styles) {
           {item.destination}
         </div>
         <div className={styles.mobilityTime}>
-          {formatRelativeMinutes(new Date(item.time))}
+          { moment(item.time).format('HH:mm')} ({ moment(item.time).fromNow() })
```

Comment:
this could be solved using javascripts Intl abstraction available via [Date.toLocaleTimeString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString?retiredLocale=de#using_options) used in our utility [`formatFriendlyTime`](https://github.com/neuland-ingolstadt/neuland.app/blob/develop/rogue-thi-app/lib/date-utils.js#L46-L57).

Reply:
Changed this in the last commit. It now uses `formatFriendlyTime` as suggested.